Vagrantfile

```diff
@@ -0,0 +1,37 @@
+unless Vagrant.has_plugin?("vagrant-hostmanager")
+    raise "vagrant-hostmanager plugin is not installed. Please install it with vagrant plugin install vagrant-hostmanager"
+end
+unless Vagrant.has_plugin?("vagrant-auto_network")
+    raise "vagrant-auto_network plugin is not installed. Please install it with vagrant plugin install vagrant-auto_network"
+end
+
+Vagrant.configure("2") do |config|
+
+  # tunables
+  env_prefix  = ENV['DRUPAL_VAGRANT_ENV_PREFIX'] || 'DRUPAL_VAGRANT'
+  project     = ENV["#{env_prefix}_PROJECT"] || 'docs'
+  # end tunables
+
+  config.vm.box         = "craychee/pantheon-drupal"
+  config.vm.box_version = ">=0.2.1"
+  path                  = "/vagrant"
+
+  config.vm.synced_folder ".", "/vagrant", :disabled => true
+  config.vm.synced_folder ".", path, :nfs => true
+  config.vm.network "private_network", :auto_network => true
+  config.hostmanager.enabled = true
+  config.hostmanager.manage_host = true
+  config.hostmanager.aliases = "#{project}.local"
+
+  config.ssh.forward_agent = true
+  config.ssh.insert_key = false
+
+  config.vm.provision :shell, inline: <<SCRIPT
+  set -ex
+  phantomjs --webdriver=8643 &> /dev/null &
+  su vagrant -c 'cd #{path} && composer install && bundle && npm install;
+  bin/sculpin generate && ln -sf #{path}/source #{path}/output_dev/.
+  bin/sculpin serve &
```

Comment:
@rachelwhitton you can do that. You would need to remove the `&` and the `disown`. If change this to `bin/sculpin serve --watch`, vagrant will just hang. This creates two problems:
1. You can't `vagrant ssh` into the machine in the same command window since it's a rolling log. Instead, you would need to open a new window.
2. If you kill the output (via ctrl-c), you won't be able to halt the vagrant machine because although you killed the output, you didn't kill the process. You would need to ssh in to kill the process before shutting down.

My recommendation is to leave it as is but update the docs: `vagrant up` will build the site for you. To work on it, you should `vagrant ssh` in, navigate to `/vagrant` and run `bin/sculpin serve --watch`. This is more aligned with vagrant's usefulness anyway: it's really good at building an environment but isn't intended to have an opinion about how you actively develop on it.